CHANGELOG.md

```diff
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+
+- Add test directory `wasm_tests`
+
+### Changed
+
+- Change CI to use `make test` instead of `cargo test`
+- Change name of feature `persistance` to the correctly spelled `persistence`
+
```

Comment:
The test folder was added in the other PR, I thought the CHANGELOG was going to be added as separate commit in a different PR (e.g. release one)
I would suggest to avoid to include the CHANGELOG change in this PR too, for consistency.